### ISSUE TITLE
Is it solved loss function?

### ISSUE THREAD

**Issue by ssb0621** [ronghuaiyang/arcface-pytorch#50]
i saw loss function error in this page
is it solved?
so if i train my own dataset, is it possible? not error on loss function?
